tests/test_mntp_trainer.py

```
@@ -364,6 +365,76 @@ def test_mntp_trainer_generates_deterministic_fallback(
     assert repeat_weights == weights
 
 
+def test_orchestrator_records_energy_metrics_for_long_running_jobs(
+    temp_dir: Path, fixed_run_uuid: UUID
```

Ruff (0.13.2)

Unused function argument: `fixed_run_uuid`

(ARG001)